frontend/src/components/widgets/Multiselect/Multiselect.tsx

```diff
@@ -62,10 +62,18 @@ interface MultiselectOption {
 class Multiselect extends React.PureComponent<Props, State> {
   private readonly formClearHelper = new FormClearHelper()
 
+  maxSelections: number = this.props.element.maxSelections
```

Comment:
Hm, does this update properly in the case that `props` changes? I'm not sure if it would, so it may be safer to just use the long-form everywhere (technically, it should be a new widget anyway because it'll have a different ID, but I'd prefer to be safer here).

Reply:
What would the long form look like?
Also, I would think that it would update but out of curiosity, why would it not?

Reply:
Long form as in just writing `this.props.element.maxSelections`

I don't see why it would update given that I'm fairly certain that unless javascript classes act weirdly, this shouldn't be any different from writing

```javascript
let maxSelections = this.props.elements.maxSelections
this.props.elements.maxSelections = 5  // Assuming this wasn't 5 before
```

and assuming that the `maxSelections` variable should change, which of course it won't